### ISSUE TITLE
UnboundLocalError: local variable 'pep_values' referenced before assignment

### ISSUE THREAD

**Issue by eeko-kon** [OpenMS/OpenMS#5926]
I just pulled the new merges from OpenMS and got this error using FeatureMapDF.get_df:

```
[~/openms-develop/openms_build/pyOpenMS/pyopenms/dataframes.py:232, in FeatureMapDF.get_df.<locals>.extract_meta_data(f)
    ]()[229](file:///Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/dataframes.py?line=228)[ else:
    ]()[230](file:///Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/dataframes.py?line=229)[     pep_values = ()
--> ]()[232](file:///Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/dataframes.py?line=231)[ yield tuple([f.getUniqueId()]) + pep_values + (f.getCharge(), f.getRT(), f.getMZ(), bb[0][0], bb[1][0], bb[0][1], bb[1][1], f.getOverallQuality(), f.getIntensity(), *vals)
```

UnboundLocalError: local variable 'pep_values' referenced before assignment]()

**Comment by jpfeuffer** [comment]
Probably another else branch missing here:
https://github.com/OpenMS/OpenMS/blob/develop/src/pyOpenMS/pyopenms/dataframes.py#L226

**Comment by axelwalter** [comment]
Exactly, the else branch was missing. Fixed in #5943
